docs/docs/cmd/spo/commandset/commandset-set.md

```diff
@@ -0,0 +1,71 @@
+# spo commandset set
+
+Updates a ListView Command Set on a site.
+
+## Usage
+
+```sh
+m365 spo commandset set [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: The site to update the ListView Command Set on.
+
+`-t, --title [title]`
+: The title of the ListView Command Set to update. Specify either `title`, `id`or `clientSideComponentId`.
+
+`-i, --id [id]`
+: The id of the ListView Command Set to update. Specify either `title`, `id`or `clientSideComponentId`.
+
+`-c, --clientSideComponentId  [clientSideComponentId]`
+: The Client Side Component Id (GUID) of the ListView Command Set to update. Specify either `title`, `id`or `clientSideComponentId`.
+
+`--newTitle [newTitle]`
+: The new title of the ListView Command Set.
+
+`-l, --listType [listType]`
+: The list or library type to register the Command Set on. Allowed values `List` or `Library`.
```

Comment:
since in this PR https://github.com/pnp/cli-microsoft365/pull/4493/files
we extended also to support SitePages it would make sense to do the same here